it/nationaldutyrepaymentcenter/stubs/FileTransferStubs.scala

```diff
@@ -1,41 +1,84 @@
 package nationaldutyrepaymentcenter.stubs
 
-
 import nationaldutyrepaymentcenter.support.WireMockSupport
 import play.api.libs.json.Json
-import uk.gov.hmrc.nationaldutyrepaymentcenter.models.{FileTransferRequest, FileTransferResult}
-
-import java.time.LocalDateTime
+import uk.gov.hmrc.nationaldutyrepaymentcenter.models.{MultiFileTransferRequest, MultiFileTransferResult}
 
 trait FileTransferStubs {
   me: WireMockSupport =>
 
-  def givenNdrcFileTransferSucceeds(fileTransferRequest: FileTransferRequest): Unit = {
+  def givenFileTransmissionsMultipleSucceeds(
+    multiFileTransferRequest: MultiFileTransferRequest = multiFileRequest(),
+    multiFileTransferResult: MultiFileTransferResult = multiFileResponse()
+  ): Unit = {
     import com.github.tomakehurst.wiremock.client.WireMock._
-    val time = LocalDateTime.of(2020, 9, 9, 0, 0)
     stubFor(
-      post(urlEqualTo("/transfer-file"))
-        .withRequestBody(equalToJson(Json.toJson(fileTransferRequest).toString(), true, true ))
+      post(urlEqualTo("/transfer-multiple-files"))
+        .withRequestBody(equalToJson(Json.toJson(multiFileTransferRequest).toString(), true, true))
         .willReturn(
           aResponse()
-            .withStatus(200)
-            .withBody(Json.toJson(FileTransferResult(fileTransferRequest.correlationId, success = true, 200, time)).toString())
+            .withStatus(202)
+            .withBody(Json.toJson(multiFileTransferResult).toString())
         )
     )
   }
 
-  def givenNdrcFileTransferFails(fileTransferRequest: FileTransferRequest): Unit = {
+  def givenFileTransmissionsMultipleFails(
+    multiFileTransferRequest: MultiFileTransferRequest = multiFileRequest()
+  ): Unit = {
     import com.github.tomakehurst.wiremock.client.WireMock._
-    val time = LocalDateTime.of(2020, 9, 9, 0, 0)
-
     stubFor(
-      post(urlEqualTo("/transfer-file"))
-        .withRequestBody(equalToJson(Json.toJson(fileTransferRequest).toString(), true, true ))
+      post(urlEqualTo("/transfer-multiple-files"))
```

Comment:
CLARIFICATION: Why are we stubbing a 409 Conflict?

Reply:
Its the same status that was previously stubbed.   I don't think it matters - any non success error status will do